src/CommunityToolkit/Xamarin.CommunityToolkit/Effects/Touch/PlatformTouchEffect.uwp.cs

```diff
@@ -172,7 +172,7 @@ void OnPointerCaptureLost(object? sender, PointerRoutedEventArgs e)
 
 		void OnPointerReleased(object? sender, PointerRoutedEventArgs e)
 		{
-			if (effect == null || effect.IsDisabled)
+			if (effect?.Element == null || effect.IsDisabled)
```

Comment:
thanks @AndreiMisiukevich for your quick response. I have cloned the repository and stepped through the code in `OnPointerReleased` which showed that after`effect.HandleTouch(TouchStatus.Completed);` the effect itself is set to `null` which leads to a NRE in `effect.HandleUserInteraction(TouchInteractionStatus.Completed);`

Reply:
after further investigation i found that `RaiseCompleted` in `TouchEffect.shared.cs` triggers a call of `OnDetached()` in `PatformTouchEffect.uwp.cs` which sets the effect to null 

https://github.com/xamarin/XamarinCommunityToolkit/blob/db8e9a28494b141295ad815dfdee44f5cc9f69a3/src/CommunityToolkit/Xamarin.CommunityToolkit/Effects/Touch/PlatformTouchEffect.uwp.cs#L95

Reply:
Thank you very much for your investigation :) I couldn't test on uwp unfortunately